lms/djangoapps/ytp/models.py

```diff
@@ -23,8 +23,8 @@ def generate_pdf(sender, instance, **kwargs):
     pdf_url = os.path.join(settings.MEDIA_URL, 'certs', pdf_filename)
 
     html_cert_url = (
-        'https://{}{}'.format(
-            settings.SITE_NAME,
+        'http://{}{}'.format(
```

Comment:
why do you change the schema to `http`?

Reply:
Https did not work locally and I forgot to bring it back. Thank you.